roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -37,3 +40,46 @@
 - set_fact:
     docker_options: "{{ openshift.docker.options | default(omit) }}"
   when: not openshift.docker.hosted_registry_insecure | default(False) | bool
+
+# Avoid docker 1.9 when installing origin < 1.2 or OSE < 3.2 on RHEL/Centos and
+# See: https://bugzilla.redhat.com/show_bug.cgi?id=1304038
+- name: Default to latest docker
+  set_fact:
+    docker_version: ''
+    docker_downgrade: False
+  
+- name: Gather installed version of common package
+  shell: >
+    rpm -q {{ openshift.common.service_type }} --queryformat '%{version}' | cut -d '.' -f1,2 | grep -v 'not installed'
+  register: installed_common_version
+  failed_when: false
+  changed_when: false
```

Comment:
On a side note, we should probably persist the installed version of OpenShift somewhere (currently we are only doing this in openshift.common.image_tag when is_containerized is true).